src/main/java/postprocess/VerticalBlur.java

```diff
@@ -1,37 +1,61 @@
 package postprocess;
 
-import graphics.Framebuffer;
 import graphics.Graphics;
 import graphics.Shader;
 import graphics.Window;
 import util.Assets;
 
+/**
+ * A Post Processing Step that renders the texture with a Horizontal Blur.
```

Comment:
This should probably be changed to "A Post Processing Step that renders the texture with a **Vertical** Blur."